### DISCUSSION TITLE
Does `pname` do anything?

### DISCUSSION THREAD

**Discussion by dpc** [ipetkov/crane#48] [Q&A] [answered]
I have a workspace, and in my `craneLib.buildPackage` steps, I can put anything as `pname = "whatever"` and it doesn't seem change what is being built.

**Comment by dpc** [comment, marked as answer]
Oh damn. I've just realized that `p` in `pname` probably stand for *Nix* `package`, not Cargo one. :facepalm:

**Reply by ipetkov** [reply]
Yep, `name`/`pname`/`version` are all consumed by `stdenv.mkDerivation` and you are free to use whatever values you wish!
